pyproject.toml

```diff
@@ -84,6 +84,8 @@ dependencies = [
     "zstandard==0.21.0",
     # License: Python Software Foundation License
     "typing-extensions==4.12.2",
+    # License:  BSD-2-Clause license 
+    "python-json-logger==2.0.7",
```

Comment:
This isn't used directly, but it allows us to specify JSON logs in our `logging.json` configuration. Useful if you are using docker, and can't add dependencies in your virtual environment.

See change in [configuration.rst] for example of how to do this.